osu.Game/Overlays/Settings/SettingsButton.cs

```diff
@@ -17,14 +18,15 @@ public SettingsButton()
             Padding = new MarginPadding { Left = SettingsPanel.CONTENT_MARGINS, Right = SettingsPanel.CONTENT_MARGINS };
         }
 
-        public string TooltipText { get; set; }
+        public LocalisableString TooltipText { get; set; }
 
         public override IEnumerable<string> FilterTerms
         {
             get
             {
-                if (TooltipText != null)
-                    return base.FilterTerms.Append(TooltipText);
+                if (TooltipText != default)
+                    // TODO: this won't work as intended once the tooltip text is translated.
+                    return base.FilterTerms.Append(TooltipText.ToString());
```

Comment:
As stated, this should be revisited when this view is being localised - but that goes for the entire `FilterTerms` mechanism of the settings sidebar.